admin/src/ConflictDetectionScannerSection.js

```diff
@@ -7,7 +7,8 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faSpinner, faCheck, faSkull } from '@fortawesome/free-solid-svg-icons'
 import { mountConflictDetectionReporter, isConflictDetectionReporterMounted } from './mountConflictDetectionReporter'
 import { __, sprintf } from '@wordpress/i18n'
-import { createInterpolateElement } from '@wordpress/element'
+import { __experimentalCreateInterpolateElement, createInterpolateElement as stableCreateInterpolateElement } from '@wordpress/element'
+const createInterpolateElement = stableCreateInterpolateElement || __experimentalCreateInterpolateElement
```

Comment:
Took me a sec but I get it. Older versions of @wordpress/element provided this as experimental. 👍

Reply:
yup.